setup.py

```diff
@@ -27,7 +27,7 @@
      'zlib1g-dev']
 ]
 
-PYSAM_INSTALLATION_COMMAND = ['pip', 'install', 'pysam>=0.15.3']
+PYSAM_INSTALLATION_COMMAND = ['pip', 'install', 'pysam<0.16.0']
```

Comment:
How about ==0.15.3 instead? we need that for UTF-8 compatibility.

Reply:
Discussed offline, modified to have both limitations.